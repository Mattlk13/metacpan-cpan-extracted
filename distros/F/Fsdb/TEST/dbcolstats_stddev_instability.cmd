#
# This input showed numeric instability in stddev,
# resulting in sqrt of a negative number.
#
# test case from hangguo
# fails with Can't take sqrt of -5.59822e-15 at /usr/share/perl5/vendor_perl/Fsdb/Filter/dbcolstats.pm line 620, <GEN0> line 106.
# under Fedora 31 with perl v5.30.1 on x86-64
#
prog='dbcolstats'
args='RCLoss_norm'
cmp='diff -c '
