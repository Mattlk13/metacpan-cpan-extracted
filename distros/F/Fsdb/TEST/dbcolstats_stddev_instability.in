#fsdb -F S Sport RCLoss_norm
9384  0.999496774037648
9384  0.999496774037648
9384  0.999496774037648
9384  0.999496774037648
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496774138983
9384  0.999496975509876
#   | csv_to_db
#   | dbrow '_Sport > 9300 && _Sport < 9400'
#   | dbsort Sport
#   | dbrow _Sport==9384
#   | dbsort -n RCLoss_norm
