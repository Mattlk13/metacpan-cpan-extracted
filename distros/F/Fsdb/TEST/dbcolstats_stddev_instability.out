#fsdb mean stddev pct_rsd conf_range conf_low conf_high conf_pct sum sum_squared min max n
0.9995 0 0 0 0.9995 0.9995 0.95 99.95 99.899 0.999496774037648 0.999496975509876 100
#   | csv_to_db
#   | dbrow '_Sport > 9300 && _Sport < 9400'
#   | dbsort Sport
#   | dbrow _Sport==9384
#   | dbsort -n RCLoss_norm
#   | dbcolstats RCLoss_norm
